File: Web/src/test/java/forms/CookieForm.java
package forms;

import aquality.selenium.elements.interfaces.IButton;
import aquality.selenium.forms.Form;
import org.openqa.selenium.By;

public class CookieForm extends Form {

    private final IButton acceptCookie = getElementFactory().getButton(By.xpath("//button[@class='button button--solid button--transparent']"), "accept cookie");


    public CookieForm() {
        super(By.xpath("//div[@class='cookies']"), "cookie form");
    }

    public void acceptCookie() {
        acceptCookie.click();
    }

}

File: Web/TestProject1/FormClasses/CookieForm.cs
using Aquality.Selenium.Elements.Interfaces;
using Aquality.Selenium.Forms;
using OpenQA.Selenium;

namespace TestProject1
{
    class CookieForm : Form
    {
        public CookieForm() : base(By.XPath("//div[@class='align__cell']//*[@class='button button--solid button--transparent']"), "Cookie Form")
        {
        }
        public IButton AcceptCookiesButton => ElementFactory.GetButton(By.XPath("//div[@class='align__cell']//*[@class='button button--solid button--transparent']"), "AcceptCookies Button");

        public void AcceptingOurCorporateOverlords()
        {
            AcceptCookiesButton.Click();
        }
    }
}
